packages/api/src/utils/acceptHeader.ts

```diff
@@ -0,0 +1,81 @@
+import {ResponseFormat} from "../interfaces.js";
+
+enum MediaType {
+  json = "application/json",
+  ssz = "application/octet-stream",
+}
+
+const MEDIA_TYPES: {
+  [K in ResponseFormat]: MediaType;
+} = {
+  json: MediaType.json,
+  ssz: MediaType.ssz,
+};
+
+function responseFormatFromMediaType(mediaType: MediaType): ResponseFormat {
+  switch (mediaType) {
+    default:
+    case MediaType.json:
+      return "json";
+    case MediaType.ssz:
+      return "ssz";
+  }
+}
+
+export function writeAcceptHeader(format?: ResponseFormat): MediaType {
+  return format == undefined ? MEDIA_TYPES["json"] : MEDIA_TYPES[format];
```

Comment:
just noticed, as format can't be `null` this should preferably be a strict equality comparison

```suggestion
  return format === undefined ? MEDIA_TYPES["json"] : MEDIA_TYPES[format];
```